### PR TITLE
Some suggestions

### PR DESCRIPTION
**Pull request by dave**

### CODE REVIEW

**Comment by dave** [comment]
Hey @nicolai86 I saw you're using jennifer! Would love to hear your thoughts and suggestions on how to improve...

I had a read through your code - it looks almost perfect! There's just a few places I might do things differently:

`Params` is supposed to be for function definitions, rather than function calls... `Call` is supposed to be for function calls... it does the same thing but I think the code reads better if they're distinct.

`For` the whole condition is supposed to go in the parameters to the `For` function... 

`Dict` for map literals this will save you lots of hassle ;)

`BlockFunc` this is down to personal preference, but I prefer to embed the logic inline with the xxxFunc methods... 

`File.Render` will give you an error - `fmt.Fprintf(file, "%#v", f)` will panic on error.

**Comment by dave** [comment]
I don't have time to look through the other files today but let me know if this is useful and I'll send another PR for the rest of the code tomorrow.

**Comment by nicolai86** [comment]
@davelondon that's a great PR I'm loving it. I actually wanted to take a more detailed look at Jennifer because I had a feeling I was misusing things but it was working okay already!

I actually like that it's not a template based code generation approach - it's the second time I'm writing a contentful generator (first was template based) and needless to say jen performs way better. Thanks so much for building this! 

Looking forward to your next PR ❤

**Comment by dave** [comment]
Oh yeah one more thing:

`Parens` renders a single item in parenthesis. Supposed to be used for type conversion or to specify evaluation order. `Call` or `Params` would be confusing in this context.

**Comment by dave** [comment]
I know "Parens" and "Params" are annoyingly similar :/

**Comment by dave** [comment]
@nicolai86 ok I'm doing another PR this evening :)

File: client.go [hunk 1]
```diff
@@ -25,58 +25,54 @@ func generateClient(f *jen.File) {
...
-		jen.For(jen.List(jen.Id("_"), jen.Id("entry"))).Op(":=").Range().Id("includes.Entries").Block(
+		jen.For(jen.List(jen.Id("_"), jen.Id("entry")).Op(":=").Range().Id("includes.Entries")).Block(
 			jen.Var().Id("included").Op("=").Lit(false),
-			jen.For(jen.List(jen.Id("_"), jen.Id("entryID"))).Op(":=").Range().Id("ids").Block(
+			jen.For(jen.List(jen.Id("_"), jen.Id("entryID")).Op(":=").Range().Id("ids")).Block(
 				jen.Id("included").Op("=").Id("included").Op("||").Id("entryID.Sys.ID").Op("==").Id("entry.Sys.ID"),
 			),
-			jen.If(jen.Id("included").Op("==").Lit(true)).Block(sts...,
-			),
+			jen.If(jen.Id("included").Op("==").Lit(true)).BlockFunc(func(g *jen.Group) {
```
Comment (nicolai86) [inline client.go:53]: really nice, makes smaller lists of statements very understandble

**Pull request merged**